app/src/main/java/org/mozilla/focus/ext/Bundle.kt

```diff
@@ -0,0 +1,73 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.focus.ext
```

Comment:
We would need all these extensions in Fenix too? Can we upstream these in a follow-up when we upgrade AC too?

Reply:
These are already in the AC PR for compiling with SDK 33. I added them in a separate commit so we can use them until the AC PR is merged.